app/ts/components/Icon/Tokens/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599.tsx

```diff
@@ -0,0 +1,22 @@
+			<path d='M6.79187 24.3911C4.69778 22.0989 3.53665 19.1063 3.53665 16.0016C3.53665 12.8968 4.69778 9.90427 6.79187 7.61204L5.91323 6.7334C3.58162 9.26247 2.28713 12.5764 2.28713 16.0162C2.28713 19.4561 3.58162 22.7699 5.91323 25.299L6.79187 24.3911Z' fill='#5A5564' />
+			<path d='M24.391 25.2024C22.0982 27.299 19.1039 28.4616 15.9971 28.4616C12.8902 28.4616 9.8959 27.299 7.60314 25.2024L6.72451 26.081C9.25357 28.4126 12.5675 29.7071 16.0073 29.7071C19.4472 29.7071 22.7611 28.4126 25.2901 26.081L24.391 25.2024Z' fill='#5A5564' />
+			<path d='M21.5471 13.0682C21.3714 11.2348 19.7899 10.6198 17.7895 10.4323V7.90771H16.2431V10.3855C15.836 10.3855 15.4201 10.3855 15.0071 10.3855V7.90771H13.4724V10.4499H10.3357V12.1047C10.3357 12.1047 11.4779 12.0842 11.4604 12.1047C11.8827 12.0582 12.2778 12.3638 12.339 12.7842V19.743C12.3209 20.0407 12.0486 20.2783 11.7532 20.2555C11.7737 20.2731 10.6286 20.2555 10.6286 20.2555L10.3357 22.1036H13.4432V24.6868H14.9896V22.1416H16.2255V24.6751H17.7748V22.1211C20.3873 21.963 22.209 21.3187 22.4375 18.8731C22.622 16.905 21.6965 16.0263 20.2175 15.6719C21.1166 15.2297 21.6731 14.4096 21.5471 13.0682ZM19.3798 18.5685C19.3798 20.4898 16.0879 20.2701 15.0394 20.2701V16.861C16.0879 16.864 19.3798 16.5623 19.3798 18.5685ZM18.6623 13.7653C18.6623 15.5226 15.9151 15.3088 15.0423 15.3088V12.2101C15.9151 12.2101 18.6623 11.9348 18.6623 13.7653Z' fill='#F09242' />
+			<path
+				d='M15.9971 32C12.8328 31.9994 9.73979 31.0606 7.10906 29.3023C4.47832 27.544 2.42801 25.0451 1.21737 22.1216C0.00673518 19.1981 -0.309871 15.9813 0.307586 12.8778C0.925042 9.77442 2.44883 6.92376 4.6863 4.6863C6.92376 2.44883 9.77442 0.925042 12.8778 0.307586C15.9813 -0.309871 19.1981 0.00673518 22.1216 1.21737C25.0451 2.42801 27.544 4.47832 29.3023 7.10906C31.0606 9.73979 31.9994 12.8328 32 15.9971C32.0004 18.0987 31.5867 20.1798 30.7826 22.1216C29.9785 24.0633 28.7998 25.8276 27.3137 27.3137C25.8276 28.7998 24.0633 29.9785 22.1216 30.7826C20.1798 31.5867 18.0987 32.0004 15.9971 32ZM15.9971 1.24767C13.0818 1.24999 10.2327 2.1165 7.8098 3.73769C5.38691 5.35888 3.49901 7.66197 2.38473 10.3559C1.27045 13.0498 0.97981 16.0135 1.54954 18.8726C2.11927 21.7316 3.5238 24.3576 5.5856 26.4185C7.6474 28.4795 10.2739 29.883 13.1332 30.4516C15.9924 31.0202 18.9561 30.7284 21.6495 29.613C24.343 28.4977 26.6453 26.6089 28.2656 24.1853C29.8858 21.7618 30.7512 18.9123 30.7523 15.9971C30.7531 14.0594 30.3719 12.1407 29.6306 10.3505C28.8893 8.56025 27.8024 6.93372 26.432 5.56388C25.0616 4.19404 23.4346 3.10776 21.6441 2.36715C19.8536 1.62654 17.9347 1.24613 15.9971 1.24767Z'
+				fill='#282138'
+			/>
+		</g>
+		<defs>
+			<clipPath id='clip0_3_125'>
+				<rect width='32' height='32' fill='white' />
+			</clipPath>
+		</defs>
+	</svg>
```

Comment:
> White text on white background is hard to read. 😆 Not a blocker though, can fix it in a future PR.
> 
> ![image](https://user-images.githubusercontent.com/886059/220840312-e412fafc-0bb8-4456-a934-2291bdae66fe.png)

This will longer be applicable in the next design update #75